CONTRIBUTING.md

```diff
@@ -299,6 +301,26 @@ And similar idea for `assert.InEpsilonSlice` / `assert.InDeltaSlice`.
 
 ---
 
+### if-err-fatal
```

Comment:
Hi!

1. I meant a more common solution than just fatal replacer.
2. `testifier` proposed to be a separate tool and maybe in separate repo

Please remove this section

Reply:
I removed the commit